multiaccounts/settings/structs.go

```diff
@@ -187,3 +188,21 @@ type Settings struct {
 	GifAPIKey                      string                        `json:"gifs/api-key"`
 	TestNetworksEnabled            bool                          `json:"test-networks-enabled?,omitempty"`
 }
+
+func (s Settings) MarshalJSON() ([]byte, error) {
+	type Alias Settings
+	item := struct {
+		Alias
+		CompressedKey string `json:"compressed-key"`
```

Comment:
Shouldn't status-go always use camel case for JSON keys?

Reply:
It should :), for historical reasons though `settings` it's using kebab case, so I kept it consistent (if you look above there's the whole struct), we should change it, but probably requires some coordinated effort with desktop, and maybe best to do in a separate pr, what do you think?

Reply:
I see now, renaming the keys would be a breaking change for Desktop. I agree, better do it in a separate PR.